tests/test_router_bgp.rb

```diff
@@ -777,7 +779,7 @@ def test_routerbgp_set_get_reconnect_interval
   def test_routerbgp_get_reconnect_interval_default
     asnum = 55
     bgp = RouterBgp.new(asnum)
-    assert_equal(60, bgp.reconnect_interval,
+    assert_equal(bgp.default_reconnect_interval, bgp.reconnect_interval,
                  "reconnect_interval should be set to default value of '60'")
```

Comment:
It should be 60 when it is supported and nil when it is not supported on the platform.